app/assets/stylesheets/dlme.scss

```diff
@@ -150,8 +152,15 @@ body {
   border-top-color: $color-gray-medium;
 }
 
-.thumbnail {
-  background-color: $color-gray-light;
+.pagination>.active>a,
```

Comment:
Can these be set using the `$pagination-*` variables?

https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss#L455

Reply:
Yeah, probably. I can't easily verify that approach works locally, since I only have one record and don't get the pagination controls, but using those variables should work (forgot they existed) so I'll redo and push up another version.